packages/sanity/src/core/create/components/StartInCreateDevInfoButton.tsx

```diff
@@ -0,0 +1,159 @@
+            <Text size={1}>
+              The Create banner is shown for new documents without initial values in Studios
+              deployed with a Create manifest.
+            </Text>
+
+            {noFallbackOrigin && (
+              <Card tone="primary" padding={2} border>
+                <Stack space={4}>
+                  <Text size={1}>
+                    The Start in Create button is disabled for development because{' '}
+                    <code>beta.create.fallbackStudioOrigin</code> has not been set in{' '}
+                    <code>sanity.config</code>.
+                  </Text>
+                  <DeployedApps studioApps={studioApp?.studioApps} />
+                </Stack>
+              </Card>
+            )}
+
+            {activeFallbackOrigin && (
+              <>
+                <Text size={1}>
+                  For development Create will use the schema from a fallback origin deployment:
+                </Text>
+                <Text size={1}>
+                  <code>{fallbackStudioOrigin}</code>
+                </Text>
+              </>
+            )}
+
+            {invalidFallbackOrigin && (
+              <Card tone="critical" padding={2} border>
+                <Stack space={4}>
+                  <Text size={1}>
+                    Fallback origin <code>{fallbackStudioOrigin}</code> set in{' '}
+                    <code>beta.create.fallbackStudioOrigin</code> is invalid.
+                  </Text>
+                  <DeployedApps studioApps={studioApp?.studioApps} />
+                </Stack>
+              </Card>
+            )}
+
+            <Text size={1}>
+              <a
+                href={'https://www.sanity.io/docs/create-content-mapping'}
+                target="_blank"
+                rel="noreferrer"
+              >
+                Content mapping for Sanity Create
+              </a>{' '}
+              describes configuration details and when the banner will appear in deployed studios.
+            </Text>
+          </Stack>
+        </WrapperCard>
+      }
+    >
+      <Button
+        ref={infoButtonRef}
+        icon={invalidFallbackOrigin ? WarningOutlineIcon : InfoOutlineIcon}
+        mode={invalidFallbackOrigin ? 'default' : 'bleed'}
+        tone={invalidFallbackOrigin ? 'critical' : 'default'}
+        onClick={toggleOpen}
+        tooltipProps={{
+          content: 'Developer info',
+        }}
+      />
+    </Popover>
+  )
+}
+
+function DeployedApps(props: {studioApps?: StudioApp[]}) {
+  const {studioApps} = props
+  if (!studioApps?.length) {
+    return <Text size={1}>There are no deployed studios available.</Text>
+  }
+
+  return (
+    <Stack space={4}>
+      <Text size={1} weight="semibold">
+        Available studio origin{studioApps.length > 1 ? 's' : ''}
+      </Text>
+      <Stack space={2}>
+        {studioApps.map((app) => {
+          return (
+            <Text key={app.id} size={1}>
+              • {app.studioUrl?.replace(`https://`, '')}
```

Comment:
• 🙈 
Ideally this was a <ul><li> thing but styling + sanity/ui stuff made it pita. Feel free to keep me honest